tests/services/events/conftest.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 CERN.
+#
+# Invenio-Records-Resources is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Pytest configuration.
+
+See https://pytest-invenio.readthedocs.io/ for documentation on which test
+fixtures are available.
+"""
+
+import pytest
+from datetime import timedelta
+from kombu import Exchange
+
+from invenio_records_resources.services.events.events import RecordEvent, \
+    RecordCreatedEvent
+
+
+@pytest.fixture(scope="module")
+def app_config(app_config):
+    """Application configuration."""
+    # handlers
+    app_config["RECORDS_RESOURCES_EVENTS_HANDLERS"] = {
```

Comment:
Most of this configuration would go to `invenio-events` and be overwritten in `invenio-app-rdm`. It is here for demo purposes.